### ISSUE TITLE
TypeError QPoint(): arguments did not match any overloaded call

### ISSUE THREAD

**Issue by x51mon** [roxgib/anki-contanki#53]
I've just updated the add-on and now I'm getting the following error. I'm using a PS3 controller.

```python
Debuginformationen:
Anki 2.1.54 (88e539aa) Python 3.10.9 Qt 6.4.2 PyQt 6.4.2
Platform: Linux
Flags: frz=False ao=True sv=2
Add-ons, last update check: 2023-02-18 12:52:17
Möglicherweise beteiligte Erweiterungen: ⁨Contanki - Controller Support for Anki beta⁩

Caught exception:
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/aqt/webview.py", line 42, in cmd
    return json.dumps(self.onCmd(str))
  File "/usr/lib/python3.10/site-packages/aqt/webview.py", line 149, in _onCmd
    return self._onBridgeCmd(str)
  File "/usr/lib/python3.10/site-packages/aqt/webview.py", line 618, in _onBridgeCmd
    handled, result = gui_hooks.webview_did_receive_js_message(
  File "/usr/lib/python3.10/site-packages/aqt/hooks_gen.py", line 4358, in __call__
    handled = filter(handled, message, context)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 124, in on_receive_message
    funcs[func](*args)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 138, in if_connected_wrapper
    func(self, *args, **kwargs)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 177, in poll
    self.do_action(state, i)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 138, in if_connected_wrapper
    func(self, *args, **kwargs)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 262, in do_action
    self.toggle_quick_select(state)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 138, in if_connected_wrapper
    func(self, *args, **kwargs)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 252, in toggle_quick_select
    self.show_quick_select(state)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 138, in if_connected_wrapper
    func(self, *args, **kwargs)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/contanki.py", line 226, in show_quick_select
    self.quick_select.appear(state)
  File "/home/simon/.local/share/Anki2/addons21/1898790263/quick.py", line 158, in appear
    for action, location in zip(self.buttons[state], self.get_geometry(state)):
  File "/home/simon/.local/share/Anki2/addons21/1898790263/quick.py", line 147, in get_geometry
    return [
  File "/home/simon/.local/share/Anki2/addons21/1898790263/quick.py", line 148, in <listcomp>
    QPoint(*self.get_cart(angle, radius, x, y))
TypeError: arguments did not match any overloaded call:
  QPoint(): too many arguments
  QPoint(xpos: int, ypos: int): argument 1 has unexpected type 'float'
  QPoint(a0: QPoint): argument 1 has unexpected type 'float'
```

**Comment by roxgib** [comment]
Not this again! Fixed with [c3e692c](https://github.com/roxgib/anki-contanki/commit/c3e692c2845b24012cc7c74676940a31bd845d8d).

This is happening because [mypy follows PEP3141](https://github.com/python/typing/issues/48) and considers it okay to pass a `float` to a function type hinted as `int`, but Qt being written in C++ does not agree. Most platforms seem to handle this by quietly silently casting the values to `int`, but I guess not everywhere. I'll just have to watch out for this in the future

**Comment by x51mon** [comment]
Thank you :)
